projects/1.16/assets/minecolonies/minecolonies/lang/zh_cn.json

```
@@ -769,155 +1202,220 @@
+  "com.minecolonies.coremod.info.builder.1.name": "限制",
+  "com.minecolonies.coremod.info.builder.1": "现在让我们了解一下建筑工人的限制：\n\n 1. 建筑工人（和大多数工人一样）小屋的等级决定可以使用的工具：\n   等级 0：木质/金质工具\n   等级 1：石头工具\n   等级 2：铁质工具\n   等级 4：钻石工具\n   等级 5：附魔工具\n\n 2. 建筑工人自动分配建筑任务的范围是 100 格，如果你想在 100 格外建造，在建筑物 UI 的右上部分，你可以选择一名建筑工人来建造该建筑",
   "com.minecolonies.coremod.info.builder.2.name": "常见问题",
-  "com.minecolonies.coremod.info.builder.2": "现在让我们解决一些频繁被问到的问题：\n\n 问题 1.  我在资源请求页上看到一个似乎无法获得的物品，或者是请求了一个必须通过精准采集附魔的工具才能获得的物品？\n   回答 1. 资源请求页面时常会出现一些诸如空气，草径等不合理的方块，这时，你可以去查看建筑工人的个人请求页，你会发现建筑工人通常会请求正确的方块，并将相应地这些方块进行转换（比如请求一个泥土随后用锹铲它）。",
+  "com.minecolonies.coremod.info.builder.2": "现在让我们了解一些常见问题：\n\n问题 1：我在资源请求页上看到一个似乎无法获得的物品，或者需要精准采集？\n回答 1：资源请求页面时常会出现一些诸如空气、草径等不合理的方块，而它们主要出现在你的建筑工人请求页，他通常会请求正确的方块并相应地进行转换（比如请求一个泥土随后用锹挖掘它）。",
   "com.minecolonies.coremod.info.builder.3.name": "升级",
   "com.minecolonies.coremod.info.builder.3": "建筑工人工作时会提升等级。\n  更高等级的建筑工人速度也会得到提升。小屋等级越高则建筑工人升级越快。\n  但是，他们的等级最高提升至小屋的等级。因此，如果他们对自己的生活环境感到不快他们便会停止升级直到你提供给他们更好的房子。",
-  "com.minecolonies.coremod.info.citizen.0.name": "建筑工人介绍",
-  "com.minecolonies.coremod.info.citizen.0": "居民小屋是你们的居民将要居住的地方。小屋的每一层都可容纳一个居民。你需要多栋房子才能容纳每个人。唯一不愿住在居民小屋里的居民是你们的卫兵。",
-  "com.minecolonies.coremod.info.citizen.1.name": "孩子们",
-  "com.minecolonies.coremod.info.citizen.1": "为了生育，一男一女居民必须住在一个居民小屋里。即使你和它们不在同一间小屋里，只要殖民地内的任何一间小屋都有一张敞开的床，孩子就可以出生。如果另一所房子里有成年人，孩子就会被他们收养，随那个成年人的姓。如果有一间没有成人的小屋，孩子会得到一个随机的名字。",
+  "com.minecolonies.coremod.info.citizen.0.name": "介绍",
+  "com.minecolonies.coremod.info.citizen.0": "住宅是你们的市民将要居住的地方。小屋的每一层都可容纳一个市民。你需要多栋房子才能容纳每个人。唯一不愿住在住宅里的市民是你们的卫兵。",
+  "com.minecolonies.coremod.info.citizen.1.name": "儿童",
+  "com.minecolonies.coremod.info.citizen.1": "为了生育，一男一女市民必须住在一个住宅里。即使你和它们不在同一间小屋里，只要殖民地内的任何一间小屋都有一张敞开的床，孩子就可以出生。如果另一所房子里有成年人，孩子就会被他们收养，随那个成年人的姓。如果有一间没有成人的小屋，孩子会得到一个随机的名字。",
   "com.minecolonies.coremod.info.citizen.2.name": "工作等级",
-  "com.minecolonies.coremod.info.citizen.2": "工人的等级由其居民小屋的等级限制。因此，如果他们对自己的居住条件不满意，他们就会停止升级，直到你为他们提供更好的住房。",
-  "com.minecolonies.coremod.info.miner.0.name": "矿工介绍",
+  "com.minecolonies.coremod.info.citizen.2": "工人的等级由其住宅的等级限制。因此，如果他们对自己的居住条件不满意，他们就会停止升级，直到你为他们提供更好的住房。",
+  "com.minecolonies.coremod.info.miner.0.name": "介绍",
   "com.minecolonies.coremod.info.miner.0": "矿工是你们殖民地的生产工人之一。他们需要一把镐来开采石块，一把锹来开采泥土和砾石，有时甚至需要一把斧，如果他们撞到矿井的话。他们还需要补给来建造自己的矿井，如木板、石板、栅栏/墙、指示牌、梯子和火把。",
-  "com.minecolonies.coremod.info.miner.1.name": "矿工限制",
-  "com.minecolonies.coremod.info.miner.1": "让我们来介绍一下限制：\n\n 1.矿工根据他们的小屋级别解锁特定的工具级别：\n LVL 0。木头/黄金工具\n LVL 1.石材工具\n LVL 2.铁制工具\n LVL 4.钻石工具\n LVL 5.附魔工具\n\n ",
+  "com.minecolonies.coremod.info.miner.1.name": "限制",
+  "com.minecolonies.coremod.info.miner.1": "让我们了解一些限制：\n\n 1. 矿工们使用的工具的等级由他们的小屋等级决定：\n   等级 0：木质/金质工具\n   等级 1：石头工具\n   等级 2：铁质工具\n   等级 4：钻石工具\n   等级 5：附魔工具",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "com.minecolonies.coremod.info.miner.1": "让我们了解一些限制：\n\n 1. 矿工们使用的工具的等级由他们的小屋等级决定：\n   等级 0：木制/金制工具\n   等级 1：石头工具\n   等级 2：铁制工具\n   等级 4：钻石工具\n   等级 5：附魔工具",
```
顺便看看其他地方有没有类似的。